core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -589,11 +589,7 @@ public enum Property {
   TSERV_COMPACTION_SERVICE_ROOT_PLANNER("tserver.compaction.major.service.root.planner",
       DefaultCompactionPlanner.class.getName(), PropertyType.CLASSNAME,
       "Compaction planner for root tablet service", "2.1.0"),
-  TSERV_COMPACTION_SERVICE_ROOT_RATE_LIMIT("tserver.compaction.major.service.root.rate.limit", "0B",
-      PropertyType.BYTES,
-      "Maximum number of bytes to read or write per second over all major"
-          + " compactions in this compaction service, or 0B for unlimited.",
-      "2.1.0"),
+  // ELASTICITY_TODO: Deprecate TSERV_COMPACTION_SERVICE_ROOT_RATE_LIMIT in 3.x
```

Comment:
Is there an issue opened for this?

Reply:
No, my plan is to do this in another PR immediately after this is merged.

Reply:
PR is #3882

Reply:
TODO markers were removed in `elasticity` when #3882 was merged into `main` and `main` merged into `elasticity`.